.github/workflows/scripts/get_label.js

```diff
@@ -0,0 +1,59 @@
+module.exports = async ({github, context, core}) => {
+    const getPullRequestNumber = (ref) => {
+        core.debug(`Parsing ref: ${ref}`);
+        // This assumes that the ref is in the form of `refs/pull/:prNumber/merge`
+        const prNumber = ref.replace(/refs\/pull\/(\d+)\/merge/, '$1');
+        return parseInt(prNumber, 10);
+    };
+  
+    const prNumber = context.issue.number || getPullRequestNumber(context.ref);
+
+    let cmakeFlags = '';
+    if(isNaN(prNumber))
+    {
+        core.setOutput("cmakeFlags", cmakeFlags);
+        return cmakeFlags;
+    }
+
+    const { data } = await github.rest.pulls.get({
+        owner: context.repo.owner,
+        repo: context.repo.repo,
+        pull_number: prNumber,
+    });
+    const labelNames = data.labels.map(label => label.name);
+
+    const labelFlags = {
+        build_all: [
+            ' -DBUILD_WITH_COLLISION_SUPPORT=ON',
+            ' -DBUILD_WITH_CASADI_SUPPORT=ON',
+            ' -DBUILD_WITH_AUTODIFF_SUPPORT=ON',
+            ' -DBUILD_WITH_CODEGEN_SUPPORT=ON',
+            ' -DBUILD_WITH_EXTRA_SUPPORT=ON',
+            ' -DBUILD_WITH_OPENMP_SUPPORT=ON',
+            ' -DBUILD_PYTHON_BINDINGS_WITH_BOOST_MPFR_SUPPORT=ON',
+            ' -DINSTALL_DOCUMENTATION=ON',
+            ' -DGENERATE_PYTHON_STUBS=ON'
+        ],
+        build_collision: ' -DBUILD_WITH_COLLISION_SUPPORT=ON',
+        build_casadi: ' -DBUILD_WITH_CASADI_SUPPORT=ON',
+        build_autodiff: ' -DBUILD_WITH_AUTODIFF_SUPPORT=ON',
+        build_codegen: ' -DBUILD_WITH_CODEGEN_SUPPORT=ON',
+        build_extra: ' -DBUILD_WITH_EXTRA_SUPPORT=ON',
+        build_openmp: ' -DBUILD_WITH_OPENMP_SUPPORT=ON',
+        build_mpfr: ' -DBUILD_PYTHON_BINDINGS_WITH_BOOST_MPFR_SUPPORT=ON',
+        build_sdf: ' -DBUILD_WITH_SDF_SUPPORT=ON'
```

Comment:
- sdf: To manage sdf we have to install gazebo dependency. But this dependency can create a lot of conflict.
- Do we add build_urdf ? In this case, we have to deactivate it by default.
- We can add build_accerelate option. CMakeLists.txt set it up to FALSE on other platform than Mac so we don't have to add a test in this script.

Reply:
We need gazebo for SDF?

Reply:
For SDF, i tried and in the ned we only need libsdformat, which can be installed via conda. It's quite light, so might be worth it. 
And i added the accelerate option

Reply:
My bad, I had some issue month ago to find sdf library on conda.